### ISSUE TITLE
New request: thejazzpianosite.com [Copyright need to request]

### ISSUE THREAD

**Issue by Rexadev** [openzim/zim-requests#1158]
- Website URL: https://www.thejazzpianosite.com/
- License: 
- Desired ZIM Title: TheJazzPianoSite.com
- Desired ZIM Description: 
- Desired ZIM Icon –png (URL or attach one): 
- Language (ISO 639-3): **eng**
- Is this a MediaWiki?: **no**

**Comment by Popolechien** [comment]
Following up on #1156 I'd advise you reach out directly to the website owner - they seem pretty cool and not after money, so there's a good chance they'd respond positively. Copy hello at kiwix.org in your email so we get a trace when/if they accept.

Other than that, the site clearly says Copyright © 2024 TJPS so this will be a `Won't fix` in a week or so.

**Comment by Rexadev** [comment]
![image](https://github.com/user-attachments/assets/985e0e4e-f7ad-4f67-aa02-b2d31ce27a7f)
@Popolechien

**Comment by Popolechien** [comment]
https://kiwix.freshdesk.com/a/tickets/76041

**Comment by RavanJAltaie** [comment]
@Popolechien the link you posted doesn't open

**Comment by Popolechien** [comment]
@RavanJAltaie Yeah I've just put it there for record keeping but Freshdesk has cut their plan from 10 to 2 agents so most of us won't have access anymore.

**Comment by benoit74** [comment]
@Popolechien please keep posting such links, even if I can't open them they are very useful for future reference IMO

**Comment by RavanJAltaie** [comment]
@Popolechien @Rexadev Have we received a response from the website owner?

**Comment by Popolechien** [comment]
Yes we're good, that's the Freshdesk ticket that no one else but me can open.

**Comment by benoit74** [comment]
Task complete but I see some Youtube videos are missing, see e.g. https://dev.library.kiwix.org/viewer#thejazzpianosite.com_en_all_2024-10/www.thejazzpianosite.com/jazz-piano-lessons/jazz-chords/polychords/ or last video on www.thejazzpianosite.com/jazz-piano-lessons/jazz-scales/melodic-minor-modes/.

I've opened an upstream bug: https://github.com/webrecorder/browsertrix-crawler/issues/699
